code/modules/mob/living/carbon/human/species.dm

```diff
@@ -164,11 +164,16 @@
 
 	if(exotic_bloodtype && C.dna.blood_type != exotic_bloodtype)
 		C.dna.blood_type = exotic_bloodtype
-
+	if(C.dna.features["legs"] == "Digitigrade Legs")
+		specflags += DIGITIGRADE
+	if(DIGITIGRADE in specflags)
+		C.Digitigrade_Leg_Swap(0)
 
 /datum/species/proc/on_species_loss(mob/living/carbon/C)
 	if(C.dna.species.exotic_bloodtype)
 		C.dna.blood_type = random_blood_type()
+	if(DIGITIGRADE in specflags)
+		C.Digitigrade_Leg_Swap(1)
```

Comment:
The proc name This_Proc_Will_Give_You_Digitigrade_Legs_If_Its_Set_To_0_And_Will_Take_Them_Away_If_Its_Set_To_1() Doesn't roll off the tongue.

Starting to remember why I didn't proc this out now